plugins/services/src/js/components/ServiceConfigDisplay.js

```diff
@@ -9,29 +9,32 @@ import ServiceConfigEnvironmentVariablesSectionDisplay from './ServiceConfigEnvi
 import ServiceConfigLabelsSectionDisplay from './ServiceConfigLabelsSectionDisplay';
 import ServiceConfigHealthChecksSectionDisplay from './ServiceConfigHealthChecksSectionDisplay';
 
+const MOUNT_TYPE = 'CreateService:ServiceConfigDisplay';
+
 // To pad priorities, so we can add components in between
 const PRIORITIES_PAD_NUMBER = 100;
 
-const SERVICE_CONFIG_DISPLAY_LIST = [
+// Register default config display components (this needs to only happen once)
+[
   ServiceConfigGeneralSectionDisplay,
   ServiceConfigNetworkingSectionDisplay,
   ServiceConfigStorageSectionDisplay,
   ServiceConfigEnvironmentVariablesSectionDisplay,
   ServiceConfigLabelsSectionDisplay,
   ServiceConfigHealthChecksSectionDisplay
-].reverse(); // MountService sorts by priority but DESC ¯\_(ツ)_/¯
-
-class ServiceConfigDisplay extends React.Component {
-  componentWillMount() {
-    SERVICE_CONFIG_DISPLAY_LIST.forEach((component, index) => {
+]
+    .forEach((component, index, componentArray) => {
```

Comment:
#nit The indentation here looks ind of funny. I would say either indent left the entire `forEach` block, or wrap it around some block.